cli/command/plugin/testdata/plugin-upgrade-terminate.golden

```diff
@@ -1,2 +1,2 @@
 Upgrading plugin foo/bar from localhost:5000/foo/bar:v0.1.0 to localhost:5000/foo/bar:v1.0.0
-Plugin images do not match, are you sure? [y/N] 
\ No newline at end of file
+Plugin images do not match, are you sure? [y/N] 
```

Comment:
Why did this change in this commit?

Reply:
It was due to what I pointed out in my review, with moving the newline printing logic to `cmd/docker/docker.go`. I suggested moving it back so that these don't change, too.